packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/index.tsx

```diff
@@ -123,7 +123,7 @@ const DeviceItem = (props: Props) => {
     const instancesWithState = props.instances.filter(i => i.state);
 
     const selectDeviceInstance = async (instance: Props['device']) => {
-        await props.selectDevice(instance);
+        props.selectDevice(instance);
```

Comment:
changes in this files are unrelated, just found some uneeded awaits and removed them